js/shared/Semgrep_js_shared.mli

```diff
@@ -60,7 +60,7 @@ val make_js_module :
   ?parse_target_ts_only:(Fpath.t -> 'a) option ->
   Language.t list ->
   (Language.t -> Fpath.t -> 'b) ->
-  (bool -> Language.t -> string -> 'c) ->
+  (Language.t -> string -> 'c) ->
```

Comment:
@ajbt200128 can we get more precise types here instead of those 'b and 'c?
It prevented to find bugs when I refactored and removed the print_errors pattern; OCaml
didn't tell me for example to update js/languages/go/Parser.ml and remove the first parameter, because
anyway in the call to make_js_module we accept something that returns a 'c and so if parameters are very general and shifted it does not catch it.